space_packet_parser/definitions.py

```diff
@@ -353,6 +353,63 @@ def _get_container_base_container(
             restrictions = []
         return self._find_container(base_container_element.attrib['containerRef']), restrictions
 
+    def _get_minimum_numpy_datatype(self, name: str, *, raw_value: bool = False) -> str | None:
```

Comment:
I'm not sure where this made the most sense... Should we actually add the `_min_dtype` to the encoding classes themselves? The issue I see is that I think it would have to be a method/property because the size_in_bits are dynamic and not known until read in.